source/features/clean-issue-filters.tsx

```diff
@@ -0,0 +1,49 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getOwnerAndRepo} from '../libs/utils';
+
+async function init(): Promise<void> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const result = await api.v4(`
+		query {
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				projects { totalCount }
+				milestones { totalCount }
+			},
+			organization(login: "${ownerName}") {
+				projects { totalCount }
+			}
+			user(login: "${ownerName}") {
+				projects { totalCount }
+			}
+		}
+	`, {
+		allowErrors: true
+	});
+
+	// Hide projects filter only if there are no repo, user, and organization level projects
+	if (
+		result.repository.projects.totalCount === 0 &&
+		(result.organization && result.organization.projects.totalCount === 0) &&
```

Comment:
If organization === undefined, the link won't be removed.

I think the check should be:

```js
!result.organization || result.organization.projects.totalCount === 0
```